sdk/src/wallet/account/operations/output_claiming.rs

```diff
@@ -162,7 +162,24 @@ impl Account {
     pub async fn claim_outputs(&self, output_ids_to_claim: Vec<OutputId>) -> crate::wallet::Result<Transaction> {
         log::debug!("[OUTPUT_CLAIMING] claim_outputs");
         let basic_outputs = self.get_basic_outputs_for_additional_inputs().await?;
-        self.claim_outputs_internal(output_ids_to_claim, basic_outputs).await
+        self.claim_outputs_internal(output_ids_to_claim, basic_outputs)
+            .await
+            .map_err(|error| {
+                // Map InsufficientStorageDepositAmount error here because it's the result of InsufficientFunds in this
+                // case and then easier to handle
+                match error {
+                    crate::wallet::Error::Block(block_error) => match *block_error {
```

Comment:
Maybe it would be better to have a manual `From` impl for the error types? Unless this should only ever happen here of course

Reply:
Initially I did in on the error type, but I think it's better to have it only here. In other places we want to keep the information that it's about the storage deposit since there it's not about the available funds, but in the claiming this is the case